src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/EditorConfig/Parsing/EditorConfigParser.cs

```diff
@@ -66,7 +66,7 @@ internal static class EditorConfigParser
 
                 // Section matching
                 var sectionMatches = s_sectionMatcher.Matches(line);
-                if (sectionMatches.Count > 0 && sectionMatches[0].Groups.Count > 0)
+                if (sectionMatches is [{ Groups.Count: > 0 }, ..])
```

Comment:
Could make this uglier 😝
```suggestion
                if (sectionMatches is [{ Groups: [..] }, ..])
```

Reply:
unfortunately, it would have to be `[{ Groups: [.., _] }, .. ])` which i actually think is too difficult to understand :D